base/applications/screensavers/3dtext/lang/pt-PT.rc

```diff
@@ -0,0 +1,18 @@
+/* Translation to Portuguese (PT) by zecarlos1957@hotmail.com 10-09-2020 */
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
```

Comment:
Why keeping neutral here ?

Reply:
Do you think it's not correct? If yes, provide some details.

Reply:
https://reactos.org/wiki/Sublang